### PR TITLE
Some leftover load csv fn unit tests 

### PR DESCRIPTION
**Pull request by ayahusseini**
- Unit tests for `load_csv_fn` added

### CODE REVIEW

File: fox_news_scraper/README.md
Comment (indiah444) [inline fox_news_scraper/README.md:1]: Thank you for updating the README with testing information, this is really helpful :)

File: fox_news_scraper/test_load_csv_fn.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    pd.testing.assert_frame_equal(args[0],  pd.DataFrame(entries))
+    assert args[1] == "test-bucket"
+    assert args[2] == "2024-10-11_12-00-00_fox_news_article_data.csv"
+
+
+@pytest.mark.parametrize("entries", [
+    ([{'col1': [1, 2], 'col2': [3, 4]}]),
+    ([{'col1': [1], 'col2': [3]}]),
+    ([({'col1': [], 'col2': []})])
+])
```
Comment (indiah444) [inline fox_news_scraper/test_load_csv_fn.py:42]: Really good use of parameterised tests

File: fox_news_scraper/test_load_csv_fn.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import pytest
+import pandas as pd
+from unittest.mock import patch, MagicMock
+
+from load_csv_fn import combine_entries_to_dataframe, process_rss_feeds_and_upload
+
+
+@patch('load_csv_fn.upload_dataframe_to_s3')
+@patch('load_csv_fn.combine_entries_to_dataframe')
+@patch('load_csv_fn.fetch_from_multiple_feeds')
```
Comment (megansam5) [inline fox_news_scraper/test_load_csv_fn.py:14]: thats a whooooollllleeeee lot of patches

**Review by megansam5 — APPROVED** [review]
shes only gone and bloody done it

**Pull request merged**